internal/cmd/ske/credentials/kubeconfig/login/login.go

```diff
@@ -0,0 +1,223 @@
+package login
+
+import (
+	"context"
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/json"
+	"encoding/pem"
+	"fmt"
+	"time"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/cache"
+	"k8s.io/client-go/rest"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/ske/client"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-sdk-go/services/ske"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+	clientauthenticationv1 "k8s.io/client-go/pkg/apis/clientauthentication/v1"
+	"k8s.io/client-go/tools/auth/exec"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+type inputModel struct {
+	ProjectId   string
+	ClusterName string
+	CacheKey    string
+}
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "login",
+		Short: "login plugin for kubectl",
+		Long:  "login plugin for kubectl to create a short-lived kubeconfig to authenticate against a STACKIT Kubernetes Engine (SKE) cluster. To get a kubeconfig to use with the login command use the 'kubeconfig create' command",
```

Comment:
> To get a kubeconfig to use with the login command use the 'kubeconfig create' command

I don't get this sentence, is that something that the user needs to do or something that is done in the background in this command?

The `kubeconfig create` command is not part of the CLI yet

Reply:
On SKE side, we will add a new endpoint, which will be used to create these "login kubeconfigs". That's something the user would need to do once, afterward kubectl will call `stackit ske kubeconfig login` directly to get new short-lived credentials for the cluster.

Reply:
`ske kubeconfig create` command extended to allow creation of "login kubeconfigs"